### PR TITLE
feat: allow setting global `format` defaults in config

### PR DESCRIPTION
**Pull request by Tragio**
Hi there 👋 

At the moment, we can only define formats for Nuxt Picture at the component level. This PR would also allow configuring the Nuxt Picture at the global level.

I'm only now getting into Nuxt Module development (kudos to @danielroe streams), so let me know if is there anything that can be done in another way or in a better way. 💪 

Thank you very much for the amazing work and community 🙏

### CODE REVIEW

File: src/runtime/components/nuxt-picture.ts [hunk 1]
```diff
@@ -29,8 +29,8 @@ export default defineComponent({
 
     type Source = { srcset?: string, src?: string, type?: string, sizes?: string }
     const sources = computed<Source[]>(() => {
-      const format = props.format || (originalFormat.value === 'svg' ? 'svg' : 'webp')
-      const formats = format.split(',')
+      const format = props.format || $img.options.format?.[0] || (originalFormat.value === 'svg' ? 'svg' : 'webp')
```
Comment (danielroe) [inline src/runtime/components/nuxt-picture.ts:32]:
```suggestion
      const format = props.format || (originalFormat.value === 'svg' ? 'svg' : ($img.options.format?.[0] || 'webp'))
```

**Comment by hartmut-co-uk** [comment]
It would be great if we also could add `quality` and apply it to `<nuxt-image>`.
ref https://github.com/nuxt/image/issues/657

For nuxt-image we'd use the first elem `format[0]`..?
(This could be a separate PR)

File: src/runtime/components/nuxt-picture.ts (hunk 1 above)
Comment (Tragio) [reply src/runtime/components/nuxt-picture.ts]: Ahh yes yes, of course. Thank you! 😄

**Review by danielroe — APPROVED** [review]

**Comment by danielroe** [comment]
@hartmut-co-uk That makes sense to me too.

**Comment by danielroe** [comment]
Yes, it's a separate PR 👍

**Comment by hartmut-co-uk** [comment]
Could you also add unit tests for the new config?

Test cases brainstormed: 
- with global config, no prop -> global config is used
  - global config, single format -> all expected image variants
  - global config, multiple formats -> all expected image variants in all formats
- with global config, with prop -> prop is used (takes precedence over global)
- global config invalid value -> Error thrown (??) || or warning logged & ignored?!?
- no global config, no prop -> default applies ('webp'?)
- global config is ignored for 'svg'

@danielroe do we have an example of setting module config to be used in unit tests...?

**Review by danielroe — APPROVED** [review]

**Comment by danielroe** [comment]
Thank you both ❤️

**Pull request merged**